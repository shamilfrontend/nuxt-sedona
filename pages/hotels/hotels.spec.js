import { shallowMount } from '@vue/test-utils'
import Component from './index'

describe('hotels', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {
      computed: {
        hotels: () => []
      }
    })
  })

  test('mounts properly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
