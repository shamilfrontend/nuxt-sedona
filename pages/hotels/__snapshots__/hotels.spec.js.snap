// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hotels should match snapshot 1`] = `
<div class="hotels-page">
  <hotel-filters-stub></hotel-filters-stub>
  <div class="hotels-page__content">
    <hotel-sortable-stub count="0"></hotel-sortable-stub>
    <section class="hotels-page__list"></section>
  </div>
</div>
`;
